.github/workflows/check-image-vulnerabilities.yml

```diff
@@ -32,17 +26,51 @@ jobs:
           } >> "$GITHUB_STEP_SUMMARY"
 
   check-image-vulnerabilities:
-    name: Checks images in version ${{ inputs.version }} for vulnerabilities
+    name: Check images in version ${{ inputs.version }} for vulnerabilities
     runs-on: ubuntu-latest
     env:
-      QUAY_BEARER_TOKEN: ${{ secrets.QUAY_RHACS_ENG_VULNERABILITY_BEARER_TOKEN }}
+      # ACS-CS dogfooding instance.
+      ROX_ENDPOINT: "acs-cii18grublkv81uil8gg.acs.rhcloud.com"
+    permissions:
+      id-token: write
```

Comment:
What are write permissions required for?

Reply:
We need it for `central-login` action. I added a comment.